### PR TITLE
April 2021 ISM updates 

### PR DESCRIPTION
**Pull request by cwesten-CyberCX**
Updates to both the CloudSystem & HybridSystem System Security Plans to incorporate changes between October 2020 and April 2021 ISM. 

- Addition of new controls has resulted in additional controls being implemented to address ISM Control #1627 - Have included references to Conditional Access Policies and MCAS - The tuning of Conditional Access Policies and MCAS remains the responsibility of the Agency.
- Incorporated 2020 IRAP reports from Azure and O365. 
- Updated references to Intune to be "Microsoft Endpoint Manager - Intune (Intune)" at the top of major sections. 
- Updated SCCM to MECM
- Updated shared responsibility matrix to include Microsoft Forms, Microsoft Whiteboard and Microsoft Planner
- Minor spelling corrections
- System Security Plan Annex excel documents to be shared out-of-band

### CODE REVIEW

File: blueprint/security/hybrid-system-security-plan.md [hunk 1]
```diff
@@ -274,7 +275,7 @@ Outsourcing is applicable to the HybridSystem as it leverages both Azure and Off
 
 #### HybridSystem compliance approach
 
-The HybridSystem leverages Microsoft Azure and Office 365, which have been IRAP assessed at PROTECTED by Shearwater. With the exception of Microsoft Defender ATP, all HybridSystem services have been IRAP assessed.
+The HybridSystem leverages Microsoft Azure and Office 365, which have been IRAP assessed at PROTECTED by CyberCX. With the exception of Microsoft Defender ATP, all HybridSystem services have been IRAP assessed.
```
Comment (zedzedzed) [inline blueprint/security/hybrid-system-security-plan.md:278]: I think you did it with the cloud native SSP: remove Defender ATP exception

**Review by zedzedzed — CHANGES_REQUESTED** [review]
Looks good. I think it'll be ready to merge once the SSP Annexes have been included. Should be fine for you to replace the ones in https://github.com/govau/desktop.gov.au/tree/develop/assets/files/security as long as they have the same filename and format.

File: blueprint/security/hybrid-system-security-plan.md [hunk 2]
```diff
@@ -150,7 +151,7 @@ Guidelines for System Monitoring | Applicable | Monitoring of HybridSystem syste
...
 Guidelines for Database Systems | Not Applicable | The Agency is responsible for database systems leveraged by the HybridSystem.
 Guidelines for Email | Applicable | The HybridSystem leverages Office 365 to provide email functionality.
-Guidelines for Networking | Applicable | The HybridSystem is designed to primarily run on-premises but also be able to use the public internet. All security controls are implemented on the endpoint devices and the Office 365 component. The Office 365 design includes a high-level network diagram showing the components that are considered in scope.
+Guidelines for Networking | Applicable | The CloudSystem is designed to run using the public internet. All security controls are implemented on the endpoint devices and the Office 365 component. The Office 365 design includes a high-level network diagram showing the components that are considered in scope. Conditional access policies and MCAS to limit access to the environment from anonymity networks. The configuration and fine tuning of these policies is the shared responsibility of the Agency and the CloudSystem.
```
[the remark this answers is not in the document]
Comment (cwesten-CyberCX) [reply blueprint/security/hybrid-system-security-plan.md]: Have updated and checked for additional incorrect references within Hybrid SSP.

File: blueprint/security/hybrid-system-security-plan.md (hunk 1 above)
Comment (cwesten-CyberCX) [reply blueprint/security/hybrid-system-security-plan.md]: Corrected and updated

File: blueprint/security/hybrid-system-security-plan.md [hunk 3]
```diff
@@ -77,19 +77,20 @@ In accordance with the requirements of the ISM, the following security documenta
...
 * DTA – Blueprint – Platform Design (August 2020)
-* DTA – Blueprint – Office 365 Design (August 2020) 
-* DTA – Hybrid Blueprint – System Security Plan (October 2020) (this document)
-* DTA – Hybrid Blueprint – System Security Plan Annex (October 2020)
+* DTA – Blueprint – Office 365 Design (August 2020)
+* DTA – Hybrid Blueprint – System Security Plan (April 2021) (this document)
+* DTA – Hybrid Blueprint – System Security Plan Annex (April 2021)
 * DTA – Hybrid Blueprint – Security Risk Management Plan (October 2020)
 * DTA – Hybrid Blueprint – Standard Operating Procedures (October 2020)
 * DTA – Hybrid Blueprint – Incident Response Plan (October 2020)
```
Comment (zedzedzed) [inline blueprint/security/hybrid-system-security-plan.md:85]: Just wondering if these documents are in scope for this review? Similarly for the cloud-native SSP.

Comment (cwesten-CyberCX) [reply blueprint/security/hybrid-system-security-plan.md:85]: Yes - will push updates once complete  (for  both Cloud-Native and Hybrid).

File: blueprint/security/hybrid-security-risk-management-plan.md [hunk 4]
```diff
@@ -345,7 +345,7 @@ An malicious insider attempts to disable cloud-based security capabilities (e.g.
 
 * HybridSystem treatments
   * Azure AD Identity Protection configured to alert on detected identity-based and sign-in risks
-  * Azure AD PIM provides JIT privileged access
+  * Azure AD Privelaged Identity Management (PIM) provides JIT privileged access
```
Comment (zedzedzed) [inline blueprint/security/hybrid-security-risk-management-plan.md:348]: Typo: Privelaged

File: blueprint/security/security-risk-management-plan.md [hunk 5]
```diff
@@ -409,7 +409,7 @@ An authorised administrator misconfigures services increasing the risk of furthe
 
 * CloudSystem treatments
   * Azure AD Identity Protection configured to alert on detected identity-based and sign-in risks
-  * Azure AD PIM provides JIT privileged access
+  * Azure AD Privelaged Identity Manager (PIM) provides Just-In-Time (JIT) privileged access
```
Comment (zedzedzed) [inline blueprint/security/security-risk-management-plan.md:412]: Typo here too

File: blueprint/security/hybrid-system-security-plan.md (hunk 3 above)
Comment (zedzedzed) [reply blueprint/security/hybrid-system-security-plan.md:85]: You should change the date on these too now that you've reviewed them

File: blueprint/security/hybrid-security-risk-management-plan.md (hunk 4 above)
Comment (cwesten-CyberCX) [reply blueprint/security/hybrid-security-risk-management-plan.md]: Update pushed

File: blueprint/security/security-risk-management-plan.md (hunk 5 above)
Comment (cwesten-CyberCX) [reply blueprint/security/security-risk-management-plan.md]: Update pushed

File: blueprint/security/hybrid-system-security-plan.md (hunk 3 above)
Comment (cwesten-CyberCX) [reply blueprint/security/hybrid-system-security-plan.md]: Update pushed

**Pull request merged**